### ISSUE TITLE
Guard agains invalid image formats for docx

### ISSUE THREAD

**Issue by xrotwang** [clld/tsammalex#13]
```
  File "/usr/venvs/tsammalex/src/tsammalex/tsammalex/adapters.py", line 263, in write
    document.add_picture(stream, width=Inches(3.5))
  File "/usr/venvs/tsammalex/local/lib/python2.7/site-packages/docx/api.py", line 86, in add_picture
    picture = self.inline_shapes.add_picture(image_path_or_stream)
  File "/usr/venvs/tsammalex/local/lib/python2.7/site-packages/docx/parts/document.py", line 210, in add_picture
    return InlineShape.new_picture(r, image_part, rId, shape_id)
  File "/usr/venvs/tsammalex/local/lib/python2.7/site-packages/docx/shape.py", line 48, in new_picture
    image_part.default_cx, image_part.default_cy, image_part.filename
  File "/usr/venvs/tsammalex/local/lib/python2.7/site-packages/docx/parts/image.py", line 35, in default_cx
    width_in_inches = px_width / horz_dpi
ZeroDivisionError: division by zero
```

**Comment by chrzyki** [comment]
Additionally, some images can't be properly embedded in *.docx, e.g. the last image on this page. http://localhost:6543/parameters/streptopeliasenegalensis

**Comment by chrzyki** [comment]
Other half (`ZeroDivisionError`) fixed in f51d3e24c1ec61816acad674cbc6221038dd6979.
